Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -33,6 +33,7 @@
   --remotion-themed-line-height: 1.2em;
   --remotion-themed-spacing: 0.4em;
   --remotion-themed-spacing-half: calc(var(--remotion-themed-spacing) * 0.5);
+  --remotion-themed-spacing-double: calc(var(--remotion-themed-spacing) * 2);
```

Comment:
I'm thinking we should make this 'large' instead of 'double' and use factor=3 for this. it's the same indent was used with the button and there, it's factor=3.